### ISSUE TITLE
English translation is needed for after Chapter3

### ISSUE THREAD

**Issue by blackapple1202** [gaoxiang12/slambook#118]
it seems that English translation is only applied on Chapter 1,2.

**Comment by gaoxiang12** [comment]
Yes, sorry I don't have enough time currently. I'll probably do it after the second edition comes out.

**Comment by saimanoj18** [comment]
So, as of now, there is no translation to english after chapter 2 ?

Actually, I am wondering if you can just use Google translate and just put a draft version. May be it can act as a reference guide.

**Comment by ndrplz** [comment]
Any news about the English translation? That would really be an immense contribution to the community
